components/eamxx/src/physics/ml_correction/ml_correction.py

```diff
@@ -8,23 +8,51 @@
     predict,
 )
 
+
 def get_ML_model(model_path):
     if model_path == "NONE":
         return None
     config = MachineLearningConfig(models=[model_path])
     model = open_model(config)
-    return model    
+    return model
+
+
+def ensure_correction_ordering(correction):
+    for key in correction:
+        correction[key] = correction[key].transpose("ncol", "z")
+    return correction
 
-def get_ML_correction(model, T_mid, qv, cos_zenith, dt):
+
+def get_ML_correction_dQ1_dQ2(model, T_mid, qv, cos_zenith, dt):
```

Comment:
I wonder if this name could be changed to `dQT_dQqv` or something like that.  I noticed that T and qv are often referred to as q1 and q2 in our ML which is a bit difficult to understand to an outside observer.  But maybe q1 and q2 are a convention?

Reply:
`dQ1` and `dQ2` are the convention for heating and moistening, and they are heavily used throughout training and diagnostics. We can change it here though I find `dQT` and `dQqv` somewhat odd?

Reply:
Ahh, okay.  That makes sense then.  Could we add a comment here then explaining that dQ1 and dQ2 are the conventions?